managed-services/transactional-email/how-to/add-mx-record-to-your-domain.mdx

```diff
@@ -24,27 +24,30 @@ This page explains how to set up an [MX](/managed-services/transactional-email/c
   - You have [added SPF and DKIM records](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain/) to your domain.
 </Message>
 
-<Message type="note">
- Adding an MX record is not mandatory but we recommend you do it to avoid getting your emails rejected.
-</Message>
+## How to add an MX record using Scaleway's MX server
 
-## How to add an MX record
+<Message type="important">
+  Scaleway provides its own MX server to improve email deliverability. However, Scaleway's MX server is a **blackhole**. This means that **any email sent back to it, will be lost with no possibility of recovery**.
+</Message>
 
 1. Add an [MX record](/managed-services/transactional-email/concepts/#mx-record) type in the root zone of your domain on your DNS zone.
 2. Enter a name of your choice in the **Name** field.
-3. Set a priority for your MX record.
+3. Set `0` as a [priority](/managed-services/transactional-email/concepts/#priority) for your MX record.
     <Message type="note">
      Priority indicates the order in which your mail servers should be tried by a mail server that is attempting to deliver emails.
+     The lowest number indicates the server with the highest priority. A `0` represents the highest priority possible.
     </Message>
-4. Enter a hostname of your choice in the **Hostname** field.
+4. Enter `blackhole.scw-tem.cloud` in the **Hostname** field.
 5. Save your changes.
 
-    <Message type="note">
-    If you do not wish to receive emails on your domain, you can [add a null MX record](#how-to-add-a-null-mx-record).
+    <Message type="important">
+    Your changes might take up to 48 hours to apply.
     </Message>
 
 ## How to add a null MX record
 
+If you do not wish to receive emails on your domain, you can add a null MX record.
```

Comment:
This IMO should be removed to avoid response from mail server that don't want to received email with a null mx